zip-xxxx.rst

```diff
@@ -0,0 +1,290 @@
+
+
+Specification
+=============
+
+All fields in this specification are encoded as little-endian.
+
+The Zcash transaction format for transaction version 6 is as follows:
+
+Transaction Format
+------------------
+
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                              | Description                                                         |
++=============================+==========================+========================================+=====================================================================+
+| **Common Transaction Fields**                                                                                                                                         |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``header``                |``uint32``                              |Contains:                                                            |
+|                             |                          |                                        |  * ``fOverwintered`` flag (bit 31, always set)                      |
+|                             |                          |                                        |  * ``version`` (bits 30 .. 0) – transaction version.                |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nVersionGroupId``       |``uint32``                              |Version group ID (nonzero).                                          |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nConsensusBranchId``    |``uint32``                              |Consensus branch ID (nonzero).                                       |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``lock_time``             |``uint32``                              |Unix-epoch UTC time or block height, encoded as in Bitcoin.          |
++-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``nExpiryHeight``         |``uint32``                              |A block height in the range {1 .. 499999999} after which             |
```

Comment:
Do we want to specify coinbase transaction heights here, or is it ok to leave it in the detailed rules?